### PR TITLE
DS-858: Add `CONTRIBUTE.md` updates (publish new package version)

### PR DESCRIPTION
**Pull request by pauzim**

### CODE REVIEW

File: packages/design-system/CONTRIBUTE.md [hunk 1]
```diff
@@ -9,14 +9,23 @@
...
+- When creating a new PR, please check if it introduces some breaking changes - if so, please add `breaking changes` label and describe briefly what changes in the PR description (e.g `propX name changes to propY name`).
+- If you want to pause your PR publishing to the next version of `taco`, please add `don't merge - next publish` label.
+
 ## Releasing new version
 
 Use [semver](https://semver.org/) while versioning package.
 
-If you want to release new version and publish to npm bump `"version": "0.0.1",` to higher in `npm-package.json` file, gitHub actions will take care of publishing package if version is greater than previous.
+- If you want to release new version and publish to npm bump `"version": "0.0.1",` to higher in `npm-package.json` file, gitHub actions will take care of publishing package if version is greater than previous.
+- Remember to keep numbering convention: if the new version introduces breaking changes, change second number of the version, if not (it's only patch to previous one), change last number.
```
Comment (olborska-elp) [inline packages/design-system/CONTRIBUTE.md:29]: Breaking changes should change second  number or first?

Semantic versioning marks first number for breaking - do we use other convention for taco?

```
MAJOR version when you make incompatible API changes
MINOR version when you add functionality in a backward compatible manner
PATCH version when you make backward compatible bug fixes
```

**Review by olborska-elp — APPROVED** [review]
Good notes for new developers 😻

File: packages/design-system/CONTRIBUTE.md (hunk 1 above)
Comment (pauzim) [reply packages/design-system/CONTRIBUTE.md:29]: @piotrekzawada we talked about second number change for breaking changes, right? 🤔

Comment (olborska-elp) [reply packages/design-system/CONTRIBUTE.md:29]: @pauzim @piotrekzawada - Paulina you're right! I forgot about our 0 version. Comment is out of date 👌

**Pull request merged**